README.md

```diff
@@ -2,6 +2,13 @@
 
 > Demo of React Hot Reloading + TypeScript
 
-```bash
-npm start
+## Install prerequisites
+```
+npm install -g webpack webpack-dev-server
```

Comment:
Don't need this so long as:
1) they are in dependencies of package.json
2) we use the `scripts` from package.json, such as `npm start`